### PR TITLE
Refactoring "DataType" initialisation with float/double to string initialisers

### PR DESCRIPTION
**Pull request by vhavrylov-ncube**
Defloating of DataType (aka Fixed64) initialization with scientific notation strings or floats/doubles

### CODE REVIEW

File: libs/ml/examples/bert_for_finetune/main.cpp [hunk 1]
```diff
@@ -93,8 +93,8 @@ int main(int ac, char **av)
   std::string classification_output =
       g.template AddNode<fetch::ml::layers::FullyConnected<TensorType>>(
           "ClassificationOutput", {cls_token_output}, config.model_dims, 2u,
-          ActivationType::SOFTMAX, RegType::NONE, static_cast<DataType>(0),
-          WeightsInitType::XAVIER_GLOROT, false);
+          ActivationType::SOFTMAX, RegType::NONE, DataType{0}, WeightsInitType::XAVIER_GLOROT,
```
Comment (baykaner) [inline libs/ml/examples/bert_for_finetune/main.cpp]: this line makes no difference

File: libs/ml/include/ml/utilities/bert_utilities.hpp [hunk 2]
```diff
@@ -55,8 +55,8 @@ struct BERTConfig
   SizeType ff_dims           = 3072u;
   SizeType vocab_size        = 30522u;
   SizeType segment_size      = 2u;
-  DataType epsilon           = static_cast<DataType>(1e-12);
-  DataType dropout_keep_prob = static_cast<DataType>(0.9);
+  DataType epsilon           = fetch::math::Type<DataType>("0.0000000000001");
```
Comment (baykaner) [inline libs/ml/include/ml/utilities/bert_utilities.hpp]: maybe I'm going blind but this looks like 1e-13 to me...?

**Review by baykaner — CHANGES_REQUESTED** [review]

File: libs/ml/include/ml/utilities/bert_utilities.hpp (hunk 2 above)
Comment (vhavrylov-ncube) [reply libs/ml/include/ml/utilities/bert_utilities.hpp]: No, you aren't )) indeed there are 13 zeroes. Corrected.

File: libs/ml/examples/bert_for_finetune/main.cpp (hunk 1 above)
Comment (vhavrylov-ncube) [reply libs/ml/examples/bert_for_finetune/main.cpp]: reverted

**Review by baykaner — DISMISSED** [review]

**Review by baykaner — APPROVED** [review]

**Pull request merged**